ci-operator/jobs/infra-periodics.yaml

```diff
@@ -1763,3 +1763,69 @@ periodics:
         - key: sa.config-updater.arm01.config
           path: kubeconfig
         secretName: build-farm-credentials
+- agent: kubernetes
+  cluster: app.ci
+  decorate: true
+  extra_refs:
+  - base_ref: master
+    org: openshift
+    repo: release
+  interval: 12h
+  labels:
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
```

Comment:
Can we add the infra labels so we get alerted when this fails?

Reply:
Yea, that was my intention too, but I removed it for now to rehearse the job with dry-run.